tools/testutils/runtests.sh

```diff
@@ -8,7 +8,7 @@ OUTPUT_DIR="$(pwd)"
 function gather_test_results() {
   # Don't immediately exit on error anymore
   set +e
-  for d in "${REPO_DIR}"/e2etests/bazel-testlogs/*; do
+  for d in "${REPO_DIR}"/bazel-testlogs/e2etests/cvd/*; do
```

Comment:
The `e2etests/cvd` part of the path was passed as parameters, it shouldn't be  hardcoded.

Reply:
renamed the tool name to `runcvde2etests.sh` this way we make it clear the tool is only meant to run the tests under `e2etests/cvd/`